myhpi/static/scss/myHPI.scss

```diff
@@ -0,0 +1,139 @@
+// 1. Include functions first (so you can manipulate colors, SVGs, calc, etc)
+@import "./bootstrap/functions";
+
+// 2. Include any default variable overrides here
+$primary: #F5631E; // #B71B31;  // FFB530

```

Comment:
The other colors are colors that work well too?

Reply:
Depends. We would need to find a solution for yellow links, as those could be hard to read. But this is tackled in other issues.
Would delete the commented colours before merging though